sql/core/src/test/resources/sql-tests/inputs/cte.sql

```diff
@@ -53,6 +53,347 @@ SELECT * FROM t;
 WITH t AS (SELECT 1 FROM non_existing_table)
 SELECT 2;
 
+-- The following tests are ported from Postgres
```

Comment:
We have many golden file test from postgres: `sql/core/src/test/resources/sql-tests/inputs/postgreSQL`

Can you check if the new tests have duplication?

Reply:
Thanks for the pointer! I've removed the duplicated test cases from PostgreSQL.